mobile/src/libs/enums/ui/app-colors.ts

```diff
@@ -0,0 +1,12 @@
+const AppColors = {
```

Comment:
enum name should be in sing form - AppColor, also rename file to app-color.enum.ts